meshtastic/tcp_interface.py

```
@@ -585,9 +601,11 @@ def _read_bytes(self, length: int) -> bytes:
         """
         sock = self.socket
         if sock is not None:
+            if isinstance(length, bool) or not isinstance(length, int) or length <= 0:
+                raise ValueError(f"length must be a positive int, got {length!r}")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)